versions.props

```
@@ -13,6 +13,7 @@ com.palantir.safe-logging:* = 3.6.0
 com.palantir.tokens:* = 3.18.0
 com.palantir.tracing:* = 6.18.0
 com.palantir.tritium:* = 0.79.0
+com.squareup.okhttp3:* = 4.11.0
```

[4.12.0 is out as of 2023-10-16](https://square.github.io/okhttp/changelogs/changelog_4x/) and bumps okio to 3.6.0 to fix [CVE-2023-3635](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-3635)

```suggestion
com.squareup.okhttp3:* = 4.12.0
```